source/tutorials/multi-boot/dual-boot-linux.rst

```diff
@@ -0,0 +1,282 @@
+:command:`systemd-boot`, which enables you to dual-boot |CL| and an existing
+Linux distro.  Although we use Ubuntu\* 19.04 Desktop as the example here, 
+these instructions also work for other distros such as Mint Linux, Kubuntu\*, Fedora\*, CentOS\*, among others.
+
+.. contents::
+   :local:
+   :depth: 1
+
+Prerequisites
+*************
+
+* Ubuntu 19.04 Desktop is already installed. 
+* There is no unallocated space available, but there is free space 
+  on the Ubuntu root partition.
+* Follow the Preliminary steps that follow below. 
+
+.. include:: ../../get-started/bare-metal-install-desktop.rst
+   :start-after: preliminary-steps-install-desktop:
+   :end-before: install-on-target-start:
+
+Install |CL| with Advanced Installation
+***************************************
+
+#. Shut down the Ubuntu OS before proceeding.
+
+#. Boot up the |CL| live desktop image, and click the penguin 
+   icon, at left, to launch the installer.
+
+#. Complete the :guilabel:`Required Options` until you reach 
+   :guilabel:`Select Installation Media`. See Figure 1.
+
+   .. figure:: ../../_figures/multi-boot/dual-boot-linux-01.png
+      :scale: 100%
+      :alt: Required options
+
+      Figure 1: Required options  
+
+#. Click :guilabel: `Select Installation Media`.  
+
+#. Select the “Advanced Installation” option. See Figure 2.  
+
+   .. figure:: ../../_figures/multi-boot/dual-boot-linux-02.png
+      :scale: 100%
+      :alt: Advanced Installation
+
+      Figure 2: Advanced Installation 
+
+#. Click :guilabel:`Partition Media` to start the GParted tool. 
+
+#. Shrink the Ubuntu root partition to free up some space for |CL|.
+
+   a. Select the Ubuntu root partition (in this example: /dev/sda2). 
+      Right-click it and select “Resize/Move”.  See Figure 3.
+
+      .. figure:: ../../_figures/multi-boot/dual-boot-linux-03.png
+         :scale: 100%
+         :alt: Ubuntu root partition
+
+         Figure 3: Ubuntu root partition
+
+   #. In the :guilabel:`New size (MiB)` field, reduce the size of the root
+      partition, leaving at least 22GB free. This allows enough space to 
+      create a swap partition (250MB) and root partition (approx. 21GB) for 
+      |CL|.  
+
+      .. note:: 
+
+         The resulting free space appears in the “Free space following 
+         (MiB)”. Click the “Resize/Move” button.  See Figure 4.
+
+      .. figure:: ../../_figures/multi-boot/dual-boot-linux-04.png
+         :scale: 100%
+         :alt: Resize Ubuntu root
+
+         Figure 4: Resize Ubuntu root
+
+   #. Click the green checkmark button to proceed. See Figure 5.  
+
+      .. figure:: ../../_figures/multi-boot/dual-boot-linux-05.png
+         :scale: 100%
+         :alt: New unallocated space
+
+         Figure 5: New unallocated space
+
+#. Share the existing EFI system partition by designating as such.
+
+#. Right-click the :guilabel:`EFI System Partition` (e.g., /dev/sda1) and 
```

Comment:
Lines 95-110 should be a sub-section of Line 93.

Reply:
Done.